hack/dataplaneParser/iptable/iptable.go

```diff
@@ -0,0 +1,114 @@
+package iptable
+
+import (
+	"os"
+	"time"
+
+	"github.com/Azure/azure-container-networking/npm/util"
+	"golang.org/x/sys/unix"
+	"k8s.io/apimachinery/pkg/util/wait"
+)
+
+type Iptables struct {
+	Name   string
```

Comment:
lets make all the variables inside start with lowercase, we want the variables to be references through helper functions on top of the defined structs.